### ISSUE TITLE
"Attempting to start a client that is already active." After Starting Client Connection on a Stopped Client.

### ISSUE THREAD

**Issue by zalkanorr** [ooonush/FishyUnityTransport#14]
As the title says, 
I'm starting a client connection like so:
```c#
...
var joinAllocation = await RelayService.Instance.JoinAllocationAsync(lobby.Data[JoinCodeKey].Value);
            
transport.SetRelayServerData(new RelayServerData(joinAllocation, "dtls"));

transport.StartConnection(false);
``` 

Then stopping it:
```c#
transport.StopConnection(false);
``` 

I then procced to make a new connection (same code as the first code block), with the same networkmanager and transport, and the error "Attempting to start a client that is already active." comes. Line 43 in ClientSocket.

Commenting the return (Line 44) makes everything work as expected!

Also calling
```c#
Lobbies.Instance.RemovePlayerAsync(ConnectedLobby.Id, playerId);
```
after stopping the connection results in this error: 
```
[Wire]: The Wire server is sending push messages related to an unknown channel: $lobby!!!xxxxxxxxxxxxxxxxxxxxxx!!!xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx!!!xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx. (replaced the ids with x's)
```

Unity 2021.3.25f1
Windows 11

**Comment by ooonush** [comment]
What version of FishyUnityTransport do you have?

**Comment by zalkanorr** [comment]
I'm using 3.6.1.Experimental.Pro

**Comment by ooonush** [comment]
Transport😅

**Comment by zalkanorr** [comment]
Oops sorry, the latest 0.3.0.

**Comment by ooonush** [comment]
Resolved in version 0.4.0
